pom.xml

```diff
@@ -442,16 +441,6 @@ under the License.
         <groupId>commons-cli</groupId>
         <artifactId>commons-cli</artifactId>
         <version>${commonsCliVersion}</version>
-        <exclusions>
-          <exclusion>
-            <groupId>commons-lang</groupId>
-            <artifactId>commons-lang</artifactId>
-          </exclusion>
-          <exclusion>
-            <groupId>commons-logging</groupId>
-            <artifactId>commons-logging</artifactId>
```

Comment:
Why did you remove the commons-logging exclusion? This is unrelated.

Reply:
Because commons-cli does not depend on any of these, this seems legacy that was never removed? Commons cli deps are only these:
```
  <dependencies>
    <dependency>
      <groupId>org.junit.jupiter</groupId>
      <artifactId>junit-jupiter-api</artifactId>
      <scope>test</scope>
    </dependency>
    <dependency>
      <groupId>org.junit.jupiter</groupId>
      <artifactId>junit-jupiter-engine</artifactId>
      <scope>test</scope>
    </dependency>
    <dependency>
      <groupId>org.junit.jupiter</groupId>
      <artifactId>junit-jupiter-params</artifactId>
      <scope>test</scope>
    </dependency>
    <dependency>
      <groupId>commons-io</groupId>
      <artifactId>commons-io</artifactId>
      <version>2.16.1</version>
      <scope>test</scope>
    </dependency>
  </dependencies>
```

Reply:
Commons Logging has been removed as a dep?